packages/cmf/src/mock/settings.js

```diff
@@ -35,7 +35,6 @@ const settings = {
 			id: 'menu:article',
 			name: 'My article',
 			icon: 'icon-article',
-			available: 'model.actions.delete',
```

Comment:
a relic of what it was intended to achieve but we don't have angular expression ^^